dnd5e/dnd5e.d.ts

```diff
@@ -200,7 +188,7 @@ declare namespace DND5e {
   type SpellSchool = 'abj' | 'con' | 'div' | 'enc' | 'evo' | 'ill' | 'nec' | 'trs';
   type Preparation = 'prepared' | 'pact' | 'always' | 'atwill' | 'innate';
 
-  type SpellLevel = 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9;
+  type SpellLevel = 0 | 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9;
   type SpellType = `spell${SpellLevel}` | 'pact';
```

Comment:
Unimportant note, why is Github's syntax highlighting leaving this as purple?

Reply:
/shrug